app.js

```diff
@@ -19,6 +19,15 @@ app.use((err, req, res, next) => {
     }
 })
 
+// app.use((err, req, res, next)=> {
+//     console.log(err)
+//     if (err.code === "23502") {
+//       res.status(400).send({msg: "Invalid request"})
+//     } else {
+//      next(err);
+//     }
+//   })
+
```

Comment:
If this isnt needed and is not being tested for then this should be removed not commented out